tracker/tracker/src/webworker/index.ts

```diff
@@ -136,13 +136,15 @@ self.onmessage = ({ data }: any): any => {
 
   if (data.type === 'auth') {
     if (!sender) {
-      throw new Error('WebWorker: sender not initialised. Received auth.')
+      console.debug('WebWorker: sender not initialised. Received auth.')
     }
     if (!writer) {
-      throw new Error('WebWorker: writer not initialised. Received auth.')
+      console.debug('WebWorker: writer not initialised. Received auth.')
+    }
+    if (sender && writer) {
```

Comment:
I think this here is pointless since we checked it on lines 138 and 141

Reply:
Addressed